integration_test/cloudbuild-integration-test.yaml

```diff
@@ -220,27 +227,33 @@
   - ${_STORE_NAME}
   - "--project=${_PROJECT}"
   - "--dataset=${_DATASET}"
-  - "--location=${_LOCATION}"  
+  - "--location=${_LOCATION}"
   - "--quiet"
   waitFor:
   - check-store-curl
   - run-find-scu
   - run-move-scu
   - run-commitment-scu
 
+- name: 'google/cloud-sdk:250.0.0'
+  id: publish-image
+  args: [ 'bash', 'integration_test/publish-images.sh', '${_IMAGEPROJECT}', "${_PUBLISH}"]
```

Comment:
I might be wrong, but I thought we needed to define substitutions in a substitutions map like here: https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values#using_user-defined_substitutions

Reply:
it done at line 259 https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/pull/32/files/f7e916518509c3ea775afdbeb9f19b623e735906#diff-98e8aebae06d145f376ebd5cf5c14035R259
or you meant something else?

Reply:
Oh I see, sorry I thought it had to be in the same build step.